app/repositories/application/OnlineTestRepository.scala

```diff
@@ -184,6 +186,21 @@ class OnlineTestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
     collection.update(query, applicationStatusBSON, upsert = false) map ( _ => () )
   }
 
+  override def removeProgressStatuses(appId: String, progressStatuses: List[ProgressStatus]): Future[Unit] = {
```

Comment:
If this method only works for PHASE1 statuses then I think we should make this clear in the method name.